README.md

```diff
@@ -82,7 +78,7 @@ For instance, using the yaml file in the examples folder:
  yaml2sbml ../examples/ode_input1.yaml ../examples/sbml_out.xml
 ```
 
-#### yaml2PEtab command
+#### yaml2petab command
```

Comment:
Not sure, but as SBML is capitalized but the tool is `yaml2sbml`, then should `yaml2petab` be lowercase here too @jvanhoefer ?

Reply:
Valid point. I would prefer to lowercase this, also in the corresponding functions. makes typing easier and is more consistent ;)